### PR TITLE
TabBar card

### PR DESCRIPTION
**Pull request by WoloxAmquinte**
## Summary
- TabBar component is created in the utils package, the pages corresponding to each menu are created

## Screenshot
TabBar

## Jira Card
<https://wolox-support.atlassian.net/browse/QAT-1840?atlOrigin=eyJpIjoiMjUzYWQ5ODZiNTkzNGIwOWFmYTNhZGM3YmZhYTllNTUiLCJwIjoiaiJ9>

### CODE REVIEW

File: src/main/java/dictionary/ConstantApp.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package dictionary;
+
+public class ConstantApp {
+
+    private ConstantApp() {
```
Comment (ffasitella) [inline src/main/java/dictionary/ConstantApp.java]: Change the name of this class to Constants so we can maintain consistency between different technologies.

File: src/main/java/utils/TabBar.java [hunk 2]
```diff
@@ -1,12 +1,21 @@
-package pages;
+package utils;
```
Comment (ffasitella) [inline src/main/java/utils/TabBar.java]: I think this is not a util but a component. A util class or method is nothing but a helper, but this TabBar class is a visual and functional component. I think you should move this to a package named `components` inside `pages`.
What do you think @WoloxAmquinte @gaboxxz ? We can discuss this later.

File: src/test/resources/features/Sample.feature [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+  Scenario: Open Swipe Menu
+    Given I am in the WebDriverIo app
+    When  I click on the "Swipe" menu
+    Then The swipe opens
+
+  Scenario: Open WebView Menu
+    Given I am in the WebDriverIo app
+    When  I click on the "Home" menu
+    Then The Home opens
```
Comment (ffasitella) [inline src/test/resources/features/Sample.feature]: This could be an Scenario Outline

**Review by ffasitella — CHANGES_REQUESTED** [review]
Left some comments

File: src/test/resources/features/Sample.feature (hunk 3 above)
Comment (WoloxAmquinte) [reply src/test/resources/features/Sample.feature]: You are right, I will make the correction

File: src/main/java/utils/TabBar.java (hunk 2 above)
Comment (WoloxAmquinte) [reply src/main/java/utils/TabBar.java]: I think it is the best, it is something visual and it is part of the pages, I will create the component package there

File: src/main/java/dictionary/ConstantApp.java (hunk 1 above)
Comment (WoloxAmquinte) [reply src/main/java/dictionary/ConstantApp.java]: Changed

**Review by ffasitella — APPROVED** [review]
Seems pretty good 👍🏽

**Pull request merged**